lib/y2lan_restart_common.pm

```diff
@@ -141,11 +150,13 @@ sub check_network_status {
     else {
         assert_script_run 'dig suse.com|grep \'status: NOERROR\'';    # test if conection and DNS is working
     }
-    assert_script_run 'cat journal.log';                              # print journal.log
-    if ($expected_status eq 'restart') {
-        assert_script_run '[ -s journal.log ]';                       # journal.log size is greater than zero (network restarted)
+    assert_script_run 'cat journal.log';
+    if (script_run('[ -s journal.log ]')) {
```

Comment:
i would have keep the comments that you removed here. not a change request but a hint: you can pass a variable output=>"this is my description " to give some details that appears in the openQA. 
Anyway in this case, the if it would be long enough and i would rather have it separated it, passing the results in a variable.
$is_size_gt_zero = script_run('[ -s journal.log ]', output=>"Check the size of journal.log");

Reply:
doing it by code is too verbose and it is simple bash, a better comment would work, like instead of talking about the size, mention that the log was written or something like that, the meaning is the important part not the size in my opinion.

Reply:
Done

Reply:
i disagree. first of all the comment above is not request for change. it is to let Sofia get know a specific feature which is quite helpful and make our life easier when we look in the openqa screens.
I disagree because to have clean and readable code is as important as to have a good comment. In other hand, a verbose approach could end to a self-explanatory code and reduce the need for long comments.
But again it is not request for change. It is up to the coder IMO